Sources/Skyflow/core/Client.swift

```diff
@@ -289,6 +316,31 @@ private class ConnectionTokenCallback: Callback {
     }
 }
 
+private class SoapConnectionTokenCallback: Callback {
```

Comment:
Why not have a constructor for ConnectionTokenCallback which takes in a bool argument to check if it is for soapConnection